recipes/unsync/meta.yaml

```diff
@@ -0,0 +1,45 @@
+{% set name = "unsync" %}
+{% set version = "1.2.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/alex-sherman/{{ name }}/archive/v{{ version }}.tar.gz
+  sha256: 7a799a21fa7f7564c55e7de781109390e47213e17f676e9b88211cb0f4ceef50
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+  skip: True  # [py<35]
```

Comment:
```suggestion
```

Reply:
We don't build less than 36 so no need for this.

Reply:
Good to know - cheers.